packages/manager/modules/iam/src/dashboard/policies/policies.styles.scss

```diff
@@ -0,0 +1,6 @@
+.policy-description-column-label {
+  max-width: 20rem;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  display: inline-block;
+}
```

Comment:
**nitpick:** Avoid using custom CSS. You can use utility classes from Bootstrap to display ellipsis

Reply:
Done! 

I had to keep the "display: inline-block" because without it, "max-width" is not having effect.